nextcloudappstore/core/forms.py

```diff
@@ -74,3 +76,31 @@ def save(self):
         app_rating.set_current_language(self._language_code)
         app_rating.comment = self.cleaned_data['comment']
         app_rating.save()
+
+
+class AppOwnershipTransferForm(Form):
+    new_owner_username = CharField(
+        max_length=User._meta.get_field('username').max_length,
+        label=_('New owner (username)'))
+
+    def __init__(self, *args, app=None, **kwargs):
```

Comment:
I'd prefer: 
```py
def __init__(self, *args, **kwargs):
    self.app = kwargs.pop('app', None)
    super().__init__(*args, **kwargs)
```

Since I'm unsure if the variable arguments work as expected